src/SimpleDiffEq.jl

```diff
@@ -6,6 +6,7 @@ using Reexport, MuladdMacro
 @reexport using DiffEqBase
 using StaticArrays
 using RecursiveArrayTools
+using LinearAlgebra
```

Comment:
where is this used?

Reply:
https://github.com/SciML/SimpleDiffEq.jl/blob/master/src/euler_maruyama.jl#L77